### PR TITLE
Add the posibility to log with LaunchpadLogRecord

### PR DESCRIPTION
**Pull request by jvanecek**

### CODE REVIEW

File: source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st [hunk 1]
```diff
@@ -7,7 +7,8 @@ Class {
 		'configurationSession',
 		'sessionPool',
 		'descriptorsBySession',
-		'descriptorSystemWasCopied'
+		'descriptorSystemWasCopied',
+		'enableLogging'
```
Comment (mtabacman) [inline source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st]: Change the variable to `loggingEnabled` or `shouldLog`

**Review by mtabacman — DISMISSED** [review]

File: source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st [hunk 2]
```diff
@@ -7,7 +7,8 @@ Class {
 		'configurationSession',
 		'sessionPool',
 		'descriptorsBySession',
-		'descriptorSystemWasCopied'
+		'descriptorSystemWasCopied',
+		'loggingEnabled'
```
Comment (fortizpenaloza) [inline source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st:11]: Is isLogginEnabled a better name?

File: source/BaselineOfSagan/BaselineOfSagan.class.st [hunk 3]
```diff
@@ -55,7 +55,11 @@ BaselineOfSagan >> setUpDependencies: spec [
 	spec
 		baseline: 'SQLite3'
 			with: [ spec repository: 'github://pharo-rdbms/Pharo-SQLite3:v2.2.0' ];
-		project: 'Glorp SQLite3' copyFrom: 'SQLite3' with: [ spec loads: 'glorp' ]
+		project: 'Glorp SQLite3' copyFrom: 'SQLite3' with: [ spec loads: 'glorp' ]. 
+		
+	spec
```
Comment (fortizpenaloza) [inline source/BaselineOfSagan/BaselineOfSagan.class.st:60]: I don't think this is right. It looks like it's time to create a new repo for the logging model.

File: source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st [hunk 4]
```diff
@@ -48,13 +49,23 @@ PooledRDBMSSessionProvider >> configureDescriptorSystemTo: aGlorpSession [
 	^ aGlorpSession
 ]
 
+{ #category : #'private-accessing' }
+PooledRDBMSSessionProvider >> createDatabaseAccessor [
+
+	| accessor |
+	accessor := DatabaseAccessor forLogin: login.
+	loggingEnabled then: [
```
Comment (fortizpenaloza) [inline source/Sagan-RDBMS/PooledRDBMSSessionProvider.class.st:57]: Extract to #configureLoggingOn: accessor

**Review by fortizpenaloza — CHANGES_REQUESTED** [review]

File: source/BaselineOfSagan/BaselineOfSagan.class.st [hunk 5]
```diff
@@ -55,7 +55,11 @@ BaselineOfSagan >> setUpDependencies: spec [
 	spec
 		baseline: 'SQLite3'
 			with: [ spec repository: 'github://pharo-rdbms/Pharo-SQLite3:v2.2.0' ];
```
Comment (gcotelli) [inline source/BaselineOfSagan/BaselineOfSagan.class.st]: There's a version 2.2.3 of SQLite3 drivers. Maybe is worth updating also.

**Review by gcotelli — CHANGES_REQUESTED** [review]
We need to update the Pharo-SQLite3 version

File: source/BaselineOfSagan/BaselineOfSagan.class.st [hunk 6]
```diff
@@ -78,7 +78,7 @@ BaselineOfSagan >> setUpPackages: spec [
 		group: 'Tests' with: 'Sagan-Core-Tests'.
 
 	spec
-		package: 'Sagan-RDBMS' with: [ spec requires: #('Glorp-Core' 'Core' 'ObjectPool-Core' 'Launchpad-Logging') ];
+		package: 'Sagan-RDBMS' with: [ spec requires: #('Glorp-Core' 'Core' 'ObjectPool-Core' 'Bell-SUnit') ];
 		group: 'RDBMS' with: #('Sagan-RDBMS').
```
Comment (gcotelli) [inline source/BaselineOfSagan/BaselineOfSagan.class.st:83]: This package shouldn't be depending on SUnit but on Deployment.

**Review by gcotelli — CHANGES_REQUESTED** [review]

File: source/BaselineOfSagan/BaselineOfSagan.class.st (hunk 6 above)
Comment (fortizpenaloza) [reply source/BaselineOfSagan/BaselineOfSagan.class.st:83]: You're right, thanks!

File: source/BaselineOfSagan/BaselineOfSagan.class.st [hunk 7]
```diff
@@ -59,6 +59,7 @@ BaselineOfSagan >> setUpDependencies: spec [
 		
 	spec 
 		baseline: 'Bell' with: [ spec repository: 'github://ba-st/Bell:v1' ];
+		project: 'Bell-Core' copyFrom: 'Bell' with: [ spec loads: 'Development' ];
```
Comment (gcotelli) [inline source/BaselineOfSagan/BaselineOfSagan.class.st]:
```suggestion
		project: 'Bell-Core' copyFrom: 'Bell' with: [ spec loads: 'Deployment' ];
```

File: source/Sagan-RDBMS-Tests/LoggingAsserter.extension.st [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+	test they appear in the right order - fortizpenaloza"
+
+	| indexes |
+
+	indexes := expectedLogEntries collect: [ :expectedLogEntry | 
+		           memoryLogger recordings detectIndex: [ :record | 
+			           record printString includesSubstring: expectedLogEntry ]
+		           ].
+
+	^ indexes = ( indexes copy sort: [ :a :b | a < b ] )
```
Comment (gcotelli) [inline source/Sagan-RDBMS-Tests/LoggingAsserter.extension.st]: I think you can use `sorted:` here that already returns a copy of the collection

File: source/Sagan-RDBMS-Tests/SingleSessionRDMBSRepositoryTest.class.st [hunk 9]
```diff
@@ -201,12 +201,15 @@ SingleSessionRDMBSRepositoryTest >> testLogging [
...
-		assertLogRecordsMatch: #( 
+		assertLogRecordsMatchInTheFollowingOrder: #( 
 		'[DEBUG] Glorp event received {"detail":"Login"}'
 		'[DEBUG] Glorp event received {"detail":"Login finished"}'
+		'[DEBUG] Glorp event received {"detail":"Begin Transaction"}'
 		'[DEBUG] Glorp event received {"detail":"INSERT INTO Extraterrestrial (sequential_number,first_name,last_name,birth_date)  VALUES (1,''Silvester'',''Stallone'',''1946-07-06'')"}'
 		'[DEBUG] Glorp event received {"detail":"Commit Transaction"}'
 		) .
+		
+	loggingAsserter stopLoggers.
```
Comment (gcotelli) [inline source/Sagan-RDBMS-Tests/SingleSessionRDMBSRepositoryTest.class.st]: Use an `ensure:` here so if the previous steps fail, it wouldn't let dangling logger instances around. Or you can also create a method like `withLoggingAsserterDo:` that wraps the asserter creation and ensures to stop loggers at the end.

**Review by gcotelli — CHANGES_REQUESTED** [review]
Now that we are using Bell, we need to rename `GlorpLoggerToLaunchpadLogRecordAdapter` to `GlorpLoggerToLogRecordAdapter`

**Review by fortizpenaloza — APPROVED** [review]

**Review by fortizpenaloza — DISMISSED** [review]

**Review by gcotelli — APPROVED** [review]

**Pull request merged**